docs/Configuration.md

```diff
@@ -29,6 +29,14 @@ This will connect to a single server on the local machine using the default redi
 var conn = ConnectionMultiplexer.Connect("redis0:6380,redis1:6380,allowAdmin=true");
 ```
 
+If you specify a serviceName in the connection string, it will trigger sentinel mode. This will connect to a sentinel server on the local machine
```

Comment:
```suggestion
If you specify a serviceName in the connection string, it will trigger sentinel mode. This example will connect to a sentinel server on the local machine
```
IMO adding "example" here would make it a lot clearer - I read this as it _always_ connects local on first pass.